src/MeshlessFV/MeshlessFVSimulation.cpp

```diff
@@ -599,18 +599,20 @@ void MeshlessFVSimulation<ndim>::PostInitialConditionsSetup(void)
     mfvneib->BuildMpiGhostTree(true, 0, ntreebuildstep, ntreestockstep, timestep, mfv);
 #endif
 
-    mfvneib->UpdateAllProperties(mfv, nbody);
```

Comment:
I'm still getting the same issue with the gadget2 MAC error with the bifurcation but I'm computing with SPH.  Does this also need to be done with the SphSimulation file also since it seems the changes here are only for the Meshless?